--- nvidia-texture-tools/src/nvcore/DefsGnucLinux.h~	2010-05-14 19:59:28.000000000 +0200
+++ nvidia-texture-tools/src/nvcore/DefsGnucLinux.h	2012-02-11 14:25:10.170869965 +0100
@@ -47,7 +47,7 @@
 
 #define restrict    __restrict__
 
-
+/*
 // Type definitions
 typedef unsigned char       uint8;
 typedef signed char         int8;
@@ -63,3 +63,4 @@
 
 // Aliases
 typedef uint32              uint;
+*/
--- nvidia-texture-tools/src/nvcore/nvcore.h~	2010-05-14 19:59:28.000000000 +0200
+++ nvidia-texture-tools/src/nvcore/nvcore.h	2012-02-11 14:29:41.590451168 +0100
@@ -99,6 +99,23 @@
 #define NV_ENDIAN_STRING	POSH_ENDIAN_STRING
 
 
+// Type definitions:
+typedef posh_u8_t   uint8;
+typedef posh_i8_t   int8;
+
+typedef posh_u16_t  uint16;
+typedef posh_i16_t  int16;
+
+typedef posh_u32_t  uint32;
+typedef posh_i32_t  int32;
+
+typedef posh_u64_t  uint64;
+typedef posh_i64_t  int64;
+
+// Aliases
+typedef uint32      uint;
+
+
 // Version string:
 #define NV_VERSION_STRING \
 	NV_OS_STRING "/" NV_CC_STRING "/" NV_CPU_STRING"/" \
--- nvidia-texture-tools/src/nvcore/DefsGnucDarwin.h~	2010-05-14 19:59:28.000000000 +0200
+++ nvidia-texture-tools/src/nvcore/DefsGnucDarwin.h	2012-02-11 14:36:32.977068794 +0100
@@ -2,7 +2,7 @@
 #error "Do not include this file directly."
 #endif
 
-#include <stdint.h> // uint8_t, int8_t, ...
+//#include <stdint.h> // uint8_t, int8_t, ...
 
 // Function linkage
 #define DLL_IMPORT
@@ -48,7 +48,7 @@
 
 #define restrict    __restrict__
 
-
+/*
 // Type definitions
 typedef uint8_t     uint8;
 typedef int8_t      int8;
@@ -64,3 +64,4 @@
 
 // Aliases
 typedef uint32      uint;
+*/
--- nvidia-texture-tools/src/nvcore/DefsGnucWin32.h~	2010-05-14 19:59:28.000000000 +0200
+++ nvidia-texture-tools/src/nvcore/DefsGnucWin32.h	2012-02-11 14:37:26.864584734 +0100
@@ -41,7 +41,7 @@
 
 #define restrict	__restrict__
 
-
+/*
 // Type definitions
 typedef unsigned char		uint8;
 typedef signed char			int8;
@@ -57,3 +57,4 @@
 
 // Aliases
 typedef uint32				uint;
+*/
--- nvidia-texture-tools/src/nvcore/DefsVcWin32.h~	2010-05-14 19:59:28.000000000 +0200
+++ nvidia-texture-tools/src/nvcore/DefsVcWin32.h	2012-02-11 14:38:15.159097685 +0100
@@ -39,7 +39,7 @@
 #define __FUNC__ __FUNCTION__ 
 #endif
 
-
+/*
 // Type definitions
 typedef unsigned char       uint8;
 typedef signed char         int8;
@@ -55,7 +55,7 @@
 
 // Aliases
 typedef uint32              uint;
-
+*/
 
 // Unwanted VC++ warnings to disable.
 /*
